_releases/3.12.0.md

```diff
@@ -0,0 +1,100 @@
+---
+version: 3.12.0
+releaseDate: 2019-01-31
+layout: release
+---
+
+## New Features <img width="25" alt ="features" src="https://user-images.githubusercontent.com/28867440/42755172-d8046d70-88ff-11e8-917c-ed1fcf2aac8c.png">
+
+- Invites and contract.availableRoles and tenantPolicy.roles. 
+All the pending invites made with an already removed role from the Contract or Tenant will be deleted.
+
+## Bug Fixes <img width="25" alt="bugfix" src="https://user-images.githubusercontent.com/28867440/42694680-e35bc0c4-86bb-11e8-8d4d-d2b1708e9213.png">
+
+- [EIOT 618] Permission assignment for OWNER role is static. 
```

Comment:
Telekom specific. But affects platform behaviour, which needs to be documented somehow.